source/includes/fact-minimum-versions.rst

```diff
@@ -2,6 +2,13 @@ Starting in version 1.2, ``mongosync`` can sync between clusters running
 some older (pre-6.0) releases of MongoDB Server. This feature requires
 additional preparation and configuration in pre-6.0 releases.
 
-If you would like to migrate pre-6.0 versions of MongoDB Server
-using ``mongosync``, please `contact <https://mongodb.com/contact>`__
-your account team to inquire about Professional Services.
+Also, ``mongosync`` can perform a cross version migration
```

Comment:
I think we don't need the change here. On the limitation page, we already have a paragraph talking about cross-version migration from `cross-version-sync.rst`